shellbase-slack/src/main/scala/com/sumologic/shellbase/slack/PostCommandToSlack.scala

```diff
@@ -27,17 +27,18 @@ trait PostCommandToSlack extends ShellCommand with PostToSlackHelper {
 
   override def postCommandToSlack(commandPath: List[String], arguments: List[String]): Option[String] = {
     try {
+      var ts: Option[String] = None
```

Comment:
We're inverting the definition of what the returned result of this method means.  That's a breaking API change.  Can we do this another way?

Reply:
I actually think this API result invert makes more sense, it would return something if we posted something to Slack successfully, instead of None. Why would we want to return something when we didn't post any command on Slack(previous behavior)?

Reply:
It may make sense, but you're breaking backward compatibility.  If we consider this to be a very important change, then we should:
1. Introduce a new case class that supports both the error message and the real result
2. Do a new major version bump

(And yes, I understand that the code, as written, can not function without this change.)